lib/reporter.coffee

```diff
@@ -1,8 +1,15 @@
-$ = require 'jquery'
 path = require 'path'
 querystring = require 'querystring'
 
-_ = require 'underscore-plus'
+extend = (target, propertyMaps...) ->
+  for propertyMap in propertyMaps
+    for key, value of propertyMap
+      target[key] = value
```

Comment:
Maybe we should put a return at the end of this function. Looks like we arent using the return list anywhere.